core/src/main/java/com/yandex/yoctodb/v1/immutable/V1DatabaseReader.java

```diff
@@ -33,6 +31,24 @@
 @ThreadSafe
 public class V1DatabaseReader extends DatabaseReader {
     private static final int DIGEST_BUF_SIZE = 4096;
+    private static final Set<Integer> SUPPORTED_FORMATS;
+
+    static {
+        Set<Integer> supported = new HashSet<>();
+        supported.add(6);
```

Comment:
Let's make a separate constant `V1DatabaseFormat.LEGACY_FORMAT`?

Reply:
I moved format version to new feature mechanism. Please, check it out, and let me know if it's OK.